tests/functional/plugincheck.js

```diff
@@ -81,3 +81,19 @@ casper.test.begin('PluginCheck, outated Firefox: ' + url, 1, function suite(test
         helpers.done();
     });
 });
+
+casper.test.begin('PluginCheck, no plugins found message: ' + url, 2, function suite(test) {
+    casper.start();
+
+    casper.userAgent(config.userAgent.fx);
+
+    casper.thenOpen(url, function() {
+        test.assertVisible('#no-plugins', 'The no plugins container is visible');
```

Comment:
Is this just assuming no plugins since it is launching with Casper? Should probably add a comment explaining this assumption if so. Also, can we be sure this will always be the case?